frontend/packages/ceph-storage-plugin/src/components/converged-credentials/credentials.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { k8sGet } from '@console/internal/module/k8s';
+import { SecretModel, ConfigMapModel } from '@console/internal/models';
+import { createModalLauncher, ModalComponentProps } from '@console/internal/components/factory';
+import { OCS_NS } from '../../constants';
+
+const parseMonEndpoints = (data: string): string => {
+  return data.split(',')[0];
+};
+
+const createJsonFile = (data: Credentials) =>
+  `data:text/json;charset=utf-8,${encodeURIComponent(JSON.stringify(data))}`;
+
+const ExportCredentials: React.FC<ModalComponentProps> = (props) => {
+  const ref = React.useRef<HTMLAnchorElement>(null);
+  const [payload, setPayload] = React.useState('');
+  React.useEffect(() => {
+    const promises = [];
+    promises.push(k8sGet(SecretModel, 'rook-ceph-mon', OCS_NS));
```

Comment:
create a constant for `rook-ceph-mon` and `rook-ceph-mon-endpoints`

Reply:
I don't think we should create consts for it. It's not reused anywhere and we will just be cluterring the namespace with consts that are only used once. If in the future we use this in multiple places then we should then refactor it IMO.

Reply:
One part of it is reusability but the other is if you don't want to touch your function when the values/names are changed. This reduces the chances of breaking the code.

if we want to reuse we need to put them in a common file but what I was saying to declare constant on top of this file itself. WYT? That will make it easy to identify the name and another person reading/changing your code don't need to look inside the code to figure out the name.